### PR TITLE
Create contributing.md

### PR DESCRIPTION
**Pull request by dan-mi-sun**
Basing off of Loomio https://github.com/loomio/loomio/blob/master/CONTRIBUTING.md

Will need to refine at a later point once we have the process sorted out.

### CODE REVIEW

**Comment by KGibb8** [comment]
Looks great, alright to leave this as a PR until it's written up properly? (i.e. links are changed, Loomio removed, etc) :smile:

**Comment by dan-mi-sun** [comment]
Yep, probably good to actually get this written up correctly. Will give it some thought and pull the contrib notes into it over the next week.

**Comment by dan-mi-sun** [comment]
@KGibb8 @sadeden Ok, have stripped the loomio comments and added in a few extras. We can refine as we go along, but helps us to meet the github community guidelines.

**Comment by dan-mi-sun** [comment]
@KGibb8 ping!

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Contributing to MMT
+
+# How do I give feedback about Loomio?
```
Comment (KGibb8) [inline contributing.md]: Nearly! :smiley:

**Comment by dan-mi-sun** [comment]
@KGibb8 should be good to go now ;)

**Pull request merged**